bindings/CMakeLists.txt

```diff
@@ -11,6 +11,14 @@ option(IDYNTREE_USES_OCTAVE "Do you want to create the OCTAVE bindings" FALSE)
 option(IDYNTREE_GENERATE_MATLAB "Enable if you have the experimental version of SWIG necessary for generating the Matlab wrapper" FALSE)
 
 find_package(SWIG)
+
+# Setup installation directory for Python bindings.
+if(IDYNTREE_PACKAGE_FOR_PYPI)
+    set(PYTHON_INSTDIR ${CMAKE_INSTALL_PREFIX})
+else()
+    set(PYTHON_INSTDIR ${IDYNTREE_PYTHON_INSTALL_DIR}/idyntree)
```

Comment:
While configuring the project, without enabling Python support, I get:

```
CMake Warning (dev) at bindings/CMakeLists.txt:19 (set):
  uninitialized variable 'IDYNTREE_PYTHON_INSTALL_DIR'
```

Reply:
This should be solved now.